src/Microsoft.ML.TensorFlow/TensorFlow/TensorflowUtils.cs

```diff
@@ -5,11 +5,23 @@
 using System;
 using System.Runtime.InteropServices;
 using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.ImageAnalytics.EntryPoints;
 
 namespace Microsoft.ML.Transforms.TensorFlow
 {
-    internal partial class TensorFlowUtils
+    public static class TensorFlowUtils
```

Comment:
>TensorFlowUtils [](start = 24, length = 15)

so in the unit test `TensorFlowTransformCifarLearningPipelineTest`  I need to call TensorFlowUtils.Initialize()  before creating the pipeline ?

Reply:
I think it would be useful, since the unit test serves as an example of how to use the transform in a learning pipeline.

---
In reply to: [214484316](https://github.com/dotnet/machinelearning/pull/795#discussion_r214484316) [](ancestors = 214484316)